backend/uclapi/uclapi/wsgi.py

```diff
@@ -28,4 +30,11 @@
 if os.environ.get("EVENTLET_NOPATCH") != 'True':
     eventlet.monkey_patch()
 
+# Download latest webpack-stats.json from S3
+# before starting WSGI
+WEBPACK_STATS_URL = "https://{}/static/webpack-stats.json".format(settings.AWS_S3_CUSTOM_DOMAIN)
```

Comment:
There is another slight issue: we run the code in parallel so it's possible that one of the threads might attempt to download the file before the others and they'll end up trampling on each other. Perhaps we need to check the file age or see if there's a lock on it before attempting to overwrite it? Otherwise we'll get nasty concurrency issues in prod.

I think at any one time we have about four Gunicorn workers per host.